Mod0/TechnicalDay2.md

```diff
@@ -0,0 +1,30 @@
+# How Humans Keep Track of Information and What That Means for Programming
+
+Like yesterday, this is an introduction designed to get you looking at code, getting familiar with syntax, and starting to think like a developer.  The expectation is that you will NOT master any topics today!
```

Comment:
I'm wondering if it would be helpful to say we're going to come back to all of these topics in Mod 1?